Utils/Dataflow/skel/stage.py

```diff
@@ -27,7 +26,16 @@
 
 
 def process(stage, message):
-    """ Single message processing. """
+    """ Single message processing.
+
+    :param stage: ETL processing stage
+    :type stage: pyDKB.dataflow.stage.ProcessorStage
+    :param message: input message with data to be processed
+    :type message: pyDKB.dataflow.communication.messages.JSONMessage
+
+    :returns: True or False in case of failure
```

Comment:
This sounds like "If there is a failure, returns True or False, I don't really know.". Maybe a comma after `True`, or something else?

Reply:
It's not my invention: http://doc.docs.sk/php/rararchive.setallowbroken.html
```
Return Values
===

Returns TRUE or FALSE on failure. It will only fail
if the file has already been closed.
```

Maybe not over-standard, but still looks fine to me.

---

We're talking about return values, so any "OR" I read as a separator between different cases. I mean, it makes no sense to read it as "(T or F) in case..." in connection with both return values:
* the return type is `bool`, so it's pretty obvious that the return value is True or False -- so why specify explicitly?
* if "(T or F) in case...", then what does it return in other cases?

By default, I tend to believe that documentation is correct and complete; if so, "T or F in case..." must cover all cases; and if some case is explicitly named, it must be related to a specific return value. Not to all of them.

It's subjective, of course, so not exactly conclusive, but... that's how I read docs, and I don't think I'm somehow unique in this respect.

Reply:
> I mean, it makes no sense to read it as "(T or F) in case..." in connection with both return values:
> 
> * the return type is `bool`, so it's pretty obvious that the return value is True or False -- so why specify explicitly?
> * if "(T or F) in case...", then what does it return in other cases?
> 
> By default, I tend to believe that documentation is correct and complete; if so, "T or F in case..." must cover all cases; and if some case is explicitly named, it must be related to a specific return value. Not to all of them.

Sure, one can (and should) apply this logic when reading, and I did it as well while reviewing, just thought that maybe we should make it a bit more obvious. But I'm not insisting, so let's leave it be.